src/burnchains/bitcoin/indexer.rs

```diff
@@ -66,7 +66,7 @@ pub const FIRST_BLOCK_TESTNET: u64 = 0;
 pub const FIRST_BLOCK_REGTEST: u64 = 0;
 
 // batch size for searching for a reorg 
-const REORG_BATCH_SIZE: u64 = 2000;
+const REORG_BATCH_SIZE: u64 = 200;
```

Comment:
Is there a reason why this was changed from 2000?  The reason it was 2000 is because (a) it's longer than the longest-ever observed reorg (by an order of magnitude) and (b) it's the largest number of headers that can be retrieved from a bitcoin node in one `getheaders` request.  Regarding (b), this means that the time difference between fetching 200 vs 2000 headers isn't that big -- it should be dominated by the network RTT.  Unless, you have benchmarks that indicate otherwise?

Reply:
Thank you for explaining the `2000` value and for being meticulous on this specific part of the PR.
As explained during our last call, I was not super comfortable patching this module and I've been optimizing for limiting my footprint. I don't have any benchmark, and we don't have any tests.
The value `200` is coming from the fact that I don't think that the code is behaving as expected (ie crashing) if the initial chain height is less than `REORG_BATCH_SIZE`.

Reply:
Do you have a stack trace for where the code crashes when the chain height is less than `REORG_BATCH_SIZE`? One option, since we aren't going to be dealing with that scenario in mainnet, is an explicit panic! if that is the case, and requiring a regtest node to play 2000 blocks before the sortitions can begin.

Reply:
Addressed with #1442/https://github.com/blockstack/stacks-blockchain/pull/1442/commits/1770a069dcef79a8b7eb29c38b2102b57af6dd6b - thank you @jcnelson.